unicoremc/constants.py

```diff
@@ -8,6 +8,12 @@
     "--paste %(config_path)s --preload"
 )
 
+UNICORECMS_MARATHON_CMD = (
+    "/var/praekelt/unicorecms-python/bin/gunicorn "
```

Comment:
@Rizziepit just a note that I'm going to update all the apps to use `unicorecms` as the virtualenv prefix